models/public/mozilla-deepspeech-0.8.2/mozilla-deepspeech-0.8.2.md

```diff
@@ -2,13 +2,13 @@
 
 ## Use Case and High-Level Description
 
-"mozilla-deepspeech-0.8.2" is a speech recognition neural network pre-trained by Mozilla
+`mozilla-deepspeech-0.8.2` is a speech recognition neural network pre-trained by Mozilla
```

Comment:
```suggestion
The `mozilla-deepspeech-0.8.2` is a speech recognition neural network pre-trained by Mozilla
```

Reply:
Done

Reply:
This is wrong; it should either be left as before or changed to "The `mozilla-deepspeech-0.8.2` model".

Reply:
Updated